src/images/icons/cursor_select.svg

```diff
@@ -0,0 +1,6 @@
+<svg viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg">
+  <title>
+    cursor select
+  </title>
+  <path d="M330.667 181.333C347.029 171.511 343.478 146.834 325.011 142.023L112.321 86.0507C96.4044 81.9041 81.9014 96.4073 86.0478 112.324L142.024 325.01C146.835 343.477 171.51 347.028 181.333 330.667L209.61 273.228L350.215 413.835C354.325 417.948 359.218 421.199 364.615 423.4C370.012 425.602 375.807 426.711 381.668 426.665C387.53 426.62 393.343 425.42 398.777 423.134C404.211 420.848 409.158 417.521 413.335 413.343C421.774 404.904 426.568 393.512 426.663 381.672C426.757 369.832 422.145 358.515 413.84 350.21L273.234 209.602L330.667 181.333Z" fill="black"/>
```

Comment:
let's remove fill from that icon, it looks it was leftover in mobile icons.

Reply:
should I remove it from the mobile one too?

Reply:
i see you did it already, thx